initializr-generator/src/main/resources/templates/docker/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM openjdk:8-jre-alpine
+MAINTAINER yaweiw@microsoft.com
+RUN apk add --update bash && rm -rf /var/cache/apk/*
+
+ADD ./packages/ /packages/
```

Comment:
Step 4/6 : ADD ./packages/ /packages/
ERROR: Service 'app' failed to build: ADD failed: stat /var/lib/docker/tmp/docker-builder586220408/packages: no such file or directory

Do we need to specify workdir?